lib/opencensus/trace/span_builder.rb

```diff
@@ -333,7 +333,7 @@ def update_stack_trace stack_trace = 0
         @stack_trace =
           case stack_trace
           when Integer
-            caller_locations(stack_trace + 2)
+            caller_locations(stack_trace + 2).map(&:to_s)
```

Comment:
One thing that confused me when writing this code was that this method doesn't appear to be called outside testing. Should I add a note about that so other developers don't get confused.